src/test/java/contacts/CreateContactTest.java

```diff
@@ -0,0 +1,105 @@
+package contacts;
+
+import base.BaseTest;
+import core.utils.DateToString;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import salesforce.config.EnvConfig;
+import salesforce.ui.pages.HomePage;
+import salesforce.ui.pages.contacts.ContactPage;
+import salesforce.ui.pages.contacts.ContactsFormPage;
+import salesforce.ui.pages.contacts.ContactsPage;
+
+import static org.testng.Assert.assertEquals;
+
+public class CreateContactTest extends BaseTest {
+
+    private HomePage homePage;
+    private ContactsPage contactsPage;
+    private ContactsFormPage contactsFormPage;
+    private ContactPage contactPage;
+    SoftAssert softAssert = new SoftAssert();
+
+    @Test
+    public void testCreateContactOnlyWithRequiredFields() {
+        String contactLastName = "Gonzales".concat(DateToString.dateToString());
+        homePage = loginPage.loginSuccessful(EnvConfig.getInstance().getUserName(), EnvConfig.getInstance().getPassword());
+        contactsPage = pageTransporter.navigateToContactsPage();
+        contactsFormPage = contactsPage.clickNewBtn();
+        contactPage = contactsFormPage.setInputField("lastName", contactLastName).clickSaveBtn();
+        softAssert.assertEquals("success\nContact " + "\"" + contactLastName + "\"" + " was created.\nClose",
+                contactPage.getTextAlertSuccess());
+        contactPage.clickDetailTab();
+//        softAssert.assertEquals(contactLastName, contactPage.getNamesText("Name"));
+        softAssert.assertEquals(contactLastName, contactPage.getPrincipalName());
+        contactsPage = pageTransporter.navigateToContactsPage();
+        softAssert.assertEquals(contactLastName,contactsPage.getNamesText(contactLastName));
+        softAssert.assertAll();
+    }
+
+
+    @Test
+    public void createContactWithAllFields() {
+        String salutation = "Dr.";
```

Comment:
Maybe you can declare this variables outside the test, so you can reuse them in other tests